Commands/general/verify.ts

```diff
@@ -0,0 +1,56 @@
+import { SlashCommandBuilder, EmbedBuilder, ChatInputCommandInteraction, User, Colors } from 'discord.js';
+import { DiscordClient } from '../../Utils/DiscordClient';
+import fs from 'fs';
+import { whisperChat } from '../../Utils/IngameChat';
+export const data = new SlashCommandBuilder()
+    .setName('verify')
+    .setDescription('Yerify yourself with the ingame bot')
+    .addStringOption((o) => o.setName(`code`).setDescription(`Put the code you recieved here.`).setRequired(true))
+
+export function execute(interaction: ChatInputCommandInteraction, client: DiscordClient) {
+        const code = interaction.options.getString("code") || "null"
+        let pending = fs.readFileSync(__dirname + "/../../Verification/pendingVerification.json","utf-8");
+        let done = fs.readFileSync(__dirname + "/../../Verification/doneVerification.json","utf-8");
+        var pendingData = JSON.parse(pending);
+        var doneData = JSON.parse(done)
+        var user = `${interaction.user.username}#${interaction.user.discriminator}`
+
+        if (pendingData.hasOwnProperty(code)){
+            const embed = new EmbedBuilder()
+            embed.setColor(Colors.Green)
+            embed.setTitle(`Ok. The account ${user} is now linked to ${pendingData[code]}`)
+            interaction.reply({
+                embeds: [embed],
+                ephemeral: true
+            })
+            if (interaction.guild === null){
+                return
+            }
+            // let verifiedRole =interaction.guild.roles.cache.find(r => r.id === client.config.guild.roles.verified)
+            // if (verifiedRole === undefined){
+            //     console.error("The verfied role doesnt exits or is not defined in the config")
+            //     return
+            // }
+            // const usr = interaction.guild?.members.cache.find(m => m.id === interaction.user.username);
+            // usr?.roles.add(verifiedRole).catch(console.error)
+
+            //Ingame Verification Message
+            const {bot} = client
+            whisperChat(bot,"Your Verification Process is done. You are now verified as " + user, pendingData[code])
+
+            doneData[user] = pendingData[code]
+            delete pendingData[code]
```

Comment:
I am slightly concerned with `delete` because if I recall correctly (I may be wrong about this) is that instead of deleting the property it actually makes the value null and while it would work it would take might take up unnecessary space if it is still in the json file

Reply:
I think it really deletes it. Because i looked into the json everytime i tested it and it was fully gone

Reply:
Ah I might be remembering things incorrectly then.